sstables/sstables.cc

```diff
@@ -1513,6 +1513,11 @@ create_sharding_metadata(schema_ptr schema, const dht::decorated_key& first_key,
     auto prange = dht::partition_range::make(dht::ring_position(first_key), dht::ring_position(last_key));
     auto sm = sharding_metadata();
     auto&& ranges = dht::split_range_to_single_shard(*schema, prange, shard).get0();
+    if (ranges.empty()) {
+        auto split_ranges_all_shards = dht::split_range_to_shards(prange, *schema);
+        sstlog.warn("create_sharding_metadata: first_key={}, last_key={}, shard={}, range={}, split_ranges_single_shard={}, split_ranges_all_shards",
+                first_key, last_key, shard, prange, ranges, split_ranges_all_shards);
```

Comment:
you missed ={} after split_ranges_all_shards. i think we can also remove the prefix split_ from split_ranges_single_shard and split_ranges_all_shards. ranges_single_shard and ranges_all_shards are clearer.

Reply:
also, there's no need for the commas...

Reply:
Fixed, thanks.